packages/gigamon/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.3.1"
+  changes:
+    - description: Fixed queries in few dashboards.
```

```suggestion
    - description: Fixed hardcoded timestamp in dashboard queries.
```
We can be more specific.(Inferred this from your commit message).

---

Updated the changelog more precisely